chatbot.html

```diff
@@ -358,39 +358,319 @@ <h2>DIETBOT</h2>
+            <li>Machine vision</li>
+            <li>Natural language processing(NLP)</li>
+            <li>Robotics</li>
+            <li>Self-driving cars</li>
+          </ul>
+          <h3>How are we implementing AI to solve our solution?</h3>
+          <p>We will leverage the use of AI to provide methods of what meals has the least carbon emissions so that our users will not spend and waste on food unneccessarily. </p>
+          <p>With this technology, less food will be wasted and the lesser amount of carbon emissions will also reduce the rate of greenhouse gases being trapped in the atmosphere and as well as minimising climate change as much as possible, with the technology of our chatbot.</p>
+          <p>Scroll down to find out more what we intend to do with our</p>
+        </div>
     </div>
+    <!-- Opening tabs function  -->
+    <script>
+    //redirects when user clicks on one of the slides and opens the tab for them
+    function gototab(redirect_id) {
+      location.replace(redirect_id);
+      opentab(event, redirect_id);
+    }
+    //sets this tab open by default when user first loads their browser
+    document.getElementById("defaultTab").click();
+    //main tab function
+    function opentab(evt, id_name) {
+      var i, tabcontent, id_name;
+
+      // Get all elements with class="tabcontent" and hide them
+      tabcontent = document.getElementsByClassName("tabcontent");
+      for (i = 0; i < tabcontent.length; i++) {
+        tabcontent[i].style.display = "none";
+        
+      }
+
+      // Get all elements with class="tablinks" and remove the class "active"
+      let tablinks = document.getElementsByClassName("tablinks");
+      for (i = 0; i < tablinks.length; i++) {
+        tablinks[i].className = tablinks[i].className.replace(" active", "");
+      }
+      
+      // Show the current tab, and add an "active" class to the button that opened the tab
+      document.querySelector(id_name).style.display = "block";
+      evt.currentTarget.className += " active";
+
+
+    }
+    </script>
+    <!-- Slide control function -->
+    <script>
+      let slideIndex = 0;
+      showSlides();
+
+      function showSlides() {
+        let i;
+        let slides = document.getElementsByClassName("mySlides");
+        let dots = document.getElementsByClassName("dot");
+        let text = document.getElementsByClassName("text");
+        for (i = 0; i < slides.length; i++) {
+          slides[i].style.display = "none"; 
+        }
+        slideIndex++;
+        if (slideIndex > slides.length) {
+          slideIndex = 1
+        }    
+        for (i = 0; i < dots.length; i++) {
+          dots[i].className = dots[i].className.replace(" active", "");
+        }
+        slides[slideIndex-1].style.display = "block";  
+        dots[slideIndex-1].className += " active";
+        setTimeout(showSlides, 3000); // Change image every 3 seconds
+
+        
+       
+      }
+    </script>
```

Comment:
Lastly, here is a JavaScript program that maintains the function of the carousel slides switching from one page to another automatically.